tests/max_encoded_len.rs

```diff
@@ -64,6 +64,13 @@ fn generic_max_length() {
 	assert_eq!(Generic::<u32>::max_encoded_len(), u32::max_encoded_len() * 2);
 }
 
+#[derive(Encode, MaxEncodedLen)]
+struct CompactField {
+	#[codec(compact)]
+	t: u64,
+	v: u64,
+}
```

Comment:
Perhaps also add a test here to ensure that the MEL for CompactField is equal to the sum of the MEL of Compact<u64> and u64?